client/state/editor/reducer.js

```diff
@@ -75,6 +75,17 @@ export function isIframeLoaded( state = false, action ) {
 	return state;
 }
 
+export function iframePort( state = null, action ) {
```

Comment:
How are the `MessagePort` objects (de)serialized? Should we ensure this isn't persisted?

Reply:
done with 2d7755be8e